pantheon-systems/documentation
# Feat: update Gatsby dependencies latest v2

To test changes on your local machine:
```
npm ci
npm run build
npm run serve
```

Pull requests should be opened in a branch off the `main` branch. For more information on contributing to Pantheon documentation, see the [Contributor Guidelines](https://pantheon.io/docs/contribute). Please refer to our [Style Guide](https://pantheon.io/docs/style-guide) and [this reference](https://developers.google.com/style) for formatting recommendations when contributing to the docs. 




Closes #

## Summary


**[Doc Page Title](https://pantheon.io/docs/doc-title)** -  <Enter a one sentence summation of the pertinent changes (including not-yet-completed work) provided by this PR.>




## Effect

The following changes are already committed:

*
*


## Remaining Work

The following changes still need to be completed:

- [ ] List any outstanding work here


--------------------------------------------------
## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)
